packages/ui/src/components/atoms/Flex.tsx

```diff
@@ -10,6 +10,8 @@ const FlexStyle = styled.div<UtilsProps>`
 
 export interface FlexProps extends UtilsProps {
   children?: ReactNode;
+  flex?: string;
+  fullWidth?: boolean; // Add fullWidth prop definition here
```

Comment:
`fullWidth` prop is not needed. You can set `width="full"`

Reply:
done